lib/logwriter.c

```diff
@@ -567,10 +567,9 @@ log_writer_emit_suppress_summary(LogWriter *self)
 
   msg_debug("Suppress timer elapsed, emitting suppression summary");
 
-  m = log_msg_new_empty();
-  m->timestamps[LM_TS_STAMP] = m->timestamps[LM_TS_RECVD];
+  m = log_msg_new_local();
   m->pri = self->last_msg->pri;
-  m->flags = LF_INTERNAL | LF_LOCAL;
+  m->flags |= LF_INTERNAL;
 
```

Comment:
There seems to be a log_msg_new_internal() function as well, which should apply to this case better.

If you update the branch with this, you have my :+1:  do go ahead and merge it.